src/rules/aggregate_eligibility_helper.py

```
@@ -0,0 +1,476 @@
+    result["expense_household_has_heating"] = has_heating
+    result["expense_household_has_dependent_care"] = has_dependent_care
+    result["expense_household_has_child_or_dependent_care"] = (
+        child_dependent_care_monthly > 0
+    )
+    
+    return result
+
+
+def compute_aggregates(request: EligibilityRequest) -> Dict[str, object]:
+
+    household = request.household[0]
```

Ruff (0.12.2)

Local variable `household` is assigned to but never used

Remove assignment to unused variable `household`

(F841)